### PR TITLE
add Chinese readme.md

### PR DESCRIPTION
**Pull request by YY0649**

### CODE REVIEW

File: README.zh.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+## FLARE_ZH: Financial Language Understanding and Prediction Evaluation Benchmark
+
+In this section, we provide a detailed performance analysis of FinMA compared to other leading models, including ChatGPT, GPT-4, lince-zero et al. For this analysis, we've chosen a range of tasks and metrics that span various aspects of financial Natural Language Processing and financial prediction.
+
+### Tasks
+
+| Data                  | Task                            | Raw     | Data Types                       | Modalities        | License           | Paper |
+| --------------------- | ------------------------------- | ------- | -------------------------------- | ----------------- | ----------------- | ----- |
+| AFQMC                 | semantic matching               | 42,511  | question data, chat              | text              | Apache-2.0        |       |
```
Comment (jiminHuang) [inline README.zh.md:102]: Why there is no reference for AFQMC and stockB?

**Review by jiminHuang — CHANGES_REQUESTED** [review]

File: README.zh.md (hunk 1 above)
Comment (YY0649) [reply README.zh.md:102]: > Why there is no reference for AFQMC and stockB?

There is no corresponding paper for these two data

Comment (jiminHuang) [reply README.zh.md:102]: Just list their website if any

File: README.zh.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+## FLARE_ZH: 金融语言理解和预测评估基准
+
+在本节中，我们将提供 FinMA 与其他领先模型（包括 ChatGPT、GPT-4、ince-zero 等）相比的详细性能分析。为了进行分析，我们选择了一系列任务和指标，涵盖了金融自然语言处理和金融预测的各个方面。
+
+### 任务
+
+| 数据                  | 任务类型                       | 原始数据| 数据类型                         | 模式              | 许可证            | 论文  |
+| --------------------- | ------------------------------- | ------- | -------------------------------- | ----------------- | ----------------- | ----- |
+| AFQMC                 | 语义匹配                        | 42,511  | 提问数据, 对话                   | 文本               | Apache-2.0        |       |
```
Comment (jiminHuang) [inline README.zh.md:261]: Add in this too

File: README.zh.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
+<p align="center" width="100%">
+<img src="https://i.postimg.cc/xTpWgq3L/pixiu-logo.png"  width="100%" height="100%">
+</p>
+
+<div>
+<div align="left">
+    <a target='_blank'>Qianqian Xie<sup>1</sup></span>&emsp;
+    <a target='_blank'>Weiguang Han<sup>1</sup></span>&emsp;
+    <a target='_blank'>Xiao Zhang<sup>2</sup></a>&emsp;
+    <a target='_blank'>Ruoyu Xiang<sup>6</sup></a>&emsp;
```
Comment (jiminHuang) [inline README.zh.md:10]: Add you and Dr. Gang's names here, and add school logo

**Review by jiminHuang — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -66,6 +66,7 @@ This repository and its contents are provided for **academic and educational pur
 
 - [English](README.md)
 - [Spainish](README.es.md)
+- [Chinese](README.zh.md)
```
Comment (jiminHuang) [inline README.md:69]: Add your name and affiliation here

File: README.zh.md [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+    <sup>4</sup>Universidad Southwest Jiaotong&emsp;
+    <sup>5</sup>Universidad de Florida&emsp;
+    <sup>6</sup>New York Universidad&emsp;
+    <sup>7</sup>Universidad de Yunnan&emsp;
+  <sup>8</sup>ChanceFocus AMC.
+</div>
+<div align="left">
+    <img src='https://i.postimg.cc/CLtkBwz7/57-EDDD9-FB0-DF712-F3-AB627163-C2-1-EF15655-13-FCA.png' alt='Wuhan University Logo' height='100px'>&emsp;
+    <img src='https://i.postimg.cc/C1XnZNK1/Sun-Yat-sen-University-Logo.png' alt='Sun Yat-Sen University Logo' height='100px'>&emsp;
+    <img src='https://i.postimg.cc/vTHJdYxN/NYU-Logo.png' alt='New York University' height='100px'>&emsp;
```
Comment (jiminHuang) [inline README.zh.md:35]:
```suggestion
     <img src='https://i.postimg.cc/V60g3hps/beb8717c1e6d7b8697274d1e5633fdcc.jpg' alt='Yunnan University Logo' height='100px'>&emsp;
```

File: README.zh.md [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+    <img src='https://i.postimg.cc/NjKhDkGY/DFAF986-CCD6529-E52-D7830-F180-D-C37-C7-DEE-4340.png' alt='Sichuan University Logo' height='100px'>&emsp;
+    <img src='https://i.postimg.cc/k5WpYj0r/SWJTULogo.png' alt='Southwest Jiaotong University Logo' height='100px'>&emsp;
+    <img src='https://i.postimg.cc/XY1s2RHD/University-of-Florida-Logo-1536x864.jpg' alt='University of Florida Logo' height='100px'>&emsp;
+    <img src='https://i.postimg.cc/V60g3hps/beb8717c1e6d7b8697274d1e5633fdcc.jpg' alt='Yunnan University Logo' height='100px'>&emsp;
```
Comment (jiminHuang) [inline README.zh.md:39]:
```suggestion
    <img src='https://i.postimg.cc/vTHJdYxN/NYU-Logo.png' alt='New York University' height='100px'>&emsp;
```

**Review by jiminHuang — CHANGES_REQUESTED** [review]

File: README.zh.md [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+| Fineval               | multiple-choice                 | 4,661   | financial exam                   | text              | Apache-2.0        | [4]   |
+| NL                    | news classification             | 10,000  | news articles                    | text              | Public            | [5]   |
+| NL2                   | news classification             | 10,000  | news articles                    | text              | Public            | [5]   |
+| NSP                   | negative news judgment          | 6,000   | news, social media text          | text              | Public            | [5]   |
+| RE                    | relationship identification     | 12,670  | news, entity pair                | text              | Public            | [5]   |
+| FE                    | sentiment analysis              | 10,000  | financial social media text      | text              | Public            | [5]   |
+| stockB                | sentiment analysis              | 270,601 | financial social media text      | text              | Apache-2.0        | [6]   |
+
+
+1. https://www.modelscope.cn/datasets/modelscope/afqmc_small/summary
```
Comment (jiminHuang) [inline README.zh.md:118]: I find the paper here
https://autonlp.ai/datasets/ant-financial-question-matching-corpus-(afqmc)-(clue-benchmark)

File: README.md [hunk 8]
```diff
@@ -7,29 +7,33 @@
...
 <div>
 <div align="left">
-    <sup>1</sup>Wuhan University&emsp;
-    <sup>2</sup>Sun Yat-Sen University&emsp;
-    <sup>3</sup>Sichuan University&emsp;
-    <sup>4</sup>Southwest Jiaotong University&emsp;
-    <sup>5</sup>University of Florida&emsp;
-    <sup>6</sup>New York Universitya&emsp;
-  <sup>7</sup>ChanceFocus AMC.
+    <sup>1</sup>Universidad de Wuhan&emsp;
```
Comment (jiminHuang) [inline README.md:23]: Why this is changed to Spanish?

File: README.zh.md [hunk 9]
```diff
@@ -0,0 +1,363 @@
...
+    <a target='_blank'>Weiguang Han<sup>1</sup></span>&emsp;
+    <a target='_blank'>Xiao Zhang<sup>2</sup></a>&emsp;
+    <a target='_blank'>Ruoyu Xiang<sup>7</sup></a>&emsp;
+    <a target='_blank'>Gang Hu<sup>4</sup></a>&emsp;
+    <a target='_blank'>Ke Qin<sup>4</sup></a>&emsp;
+    <a target='_blank'>Duanyu Feng<sup>3</sup></a>&emsp;
+    <a target='_blank'>Yongfu Dai<sup>3</sup></a>&emsp;
+    <a target='_blank'>Hao Wang<sup>3</sup></a>&emsp;
+    <a target='_blank'>Yanzhao Lai<sup>5</sup></a>&emsp;
+    <a target='_blank'>Min Peng<sup>1</sup></a>&emsp;
```
Comment (jiminHuang) [inline README.zh.md:17]: Everytime you changed something in README.md, you should also notice that we should do the same change here

**Review by jiminHuang — CHANGES_REQUESTED** [review]

**Review by jiminHuang — APPROVED** [review]

**Pull request merged**